xocto/fields/postgres/ranges.py

```diff
@@ -233,14 +233,3 @@ def value_to_string(self, obj: models.Model) -> Optional[str]:
                 "end": base_field.value_to_string(end),
             }
         )
-
-    def _is_half_finite_datetime_range(self, value: Any) -> bool:
```

Comment:
♻️ Moved to `xocto/ranges` -> https://github.com/octoenergy/xocto/pull/189#discussion_r1933268951